### ISSUE TITLE
About STARSS23 dataset using data-efficient fine tune problem.

### ISSUE THREAD

**Issue by 4A1BF2A6** [Jinbo-Hu/PSELDNets#2]
Dear Mr.Hu, thank you for publishing all the code and data, it is very helpful. When I want to perform data-efficient fine-tuning on the STARSS23 Task 3, there is no corresponding script available, so I wrote one myself. However, the metrics I obtained differ significantly from those reported in the paper. I have placed the script below. Could you please check if there are any modifications needed?

[finetune_maccdoa_adapterbit.txt](https://github.com/user-attachments/files/20139276/finetune_maccdoa_adapterbit.txt)

**Comment by Jinbo-Hu** [comment]
Maybe it can be attributed to the lack of scenario-specific synthetic datasets for STARSS23, which are not released at present. Experimental results presented in Table VIII (Results of data-efficient fine-tuning on the STARSS23 dataset) and Fig. 8 of the manuscript utilize the scenario-specific synthetic dataset. You can refer to Sec. VI-C-2 and Sec. VI-C-3 of the manuscript. 

You may create your own scenario-specific dataset using [this tools](https://github.com/Jinbo-Hu/SELD-Data-Generator) to perform data-efficient fine-tuning on the STARSS23 dataset.

**Comment by 4A1BF2A6** [comment]
Thank you for your reply. It was very helpful.

I have another question. In your paper, you mentioned, “Considering that there are no polyphonic indications in each audio clip of STARSS23 and DCASE 2021 Task 3, we constructed task-specific datasets for these tasks as well as for L3DAS22 Task 2. Each dataset consists of 120 one-minute FOA audio clips generated using simulated SRIRs.”&nbsp;

I don’t quite understand why you would test on a synthesized dataset for a specific scenario. The paper mentions that there are no polyphonic labels in each audio clip of STARSS23 and DCASE 2021 Task 3.&nbsp; Why not directly test on the STARSS23 dataset?






1243345574
***@***.***

**Comment by Jinbo-Hu** [comment]
Sec. VI-C-2 presents the results of data-efficient fine-tuning using low-resource data, which refers to small synthetic datasets using only simulated SRIRs and no data augmentation technique, as mentioned at the beginning of Sec. VI-C. Considering that there are no polyphonic indications in each audio clip of STARSS23 and DCASE 2021 Task 3, we constructed task-specific datasets for these tasks as well as for L3DAS22 Task 2. We mainly compare the results of synthetic FOA-format data with **those of corresponding pseudo-FOA-format** data in Sec. VI-C-2. The synthetic data only contains non-overlapping sound events to satisfy the single-source clip requirement described in Sec. II-A-3. 

We present the results of data-efficient fine-tuning using rich-resource data in Sec. VI-C-3. Rich-resource data refers to substantial samples that have been augmented and derived from either real-world scenes or synthesis using collected SRIRs. Notably, Sec. VI-C-3 evaluates the performance of AdapterBit on varying proportions of the training set. When employing STARSS23, a small-sized synthesized dataset comprising spatial sound events of all corresponding classes is additionally utilized, because clips from any individual rooms in STARSS23 only encompass partial sound event classes. In this Section, we evaluate AdapterBit using the training set of STARSS23. More specifically, for experiments on STARSS23, the training set includes the STARSS23 training set and a minimal synthetic dataset, while the test set is the STARSS23 validation set. As shown in Fig. 8 (c), AdapterBit may fall behind the Fine-tune method when directly trained on the STARSS23 training set. This performance disparity can be attributed to the substantial distributional differences between the simulated environments and real-world acoustic environments, which AdapterBit may not effectively learn due to its limited trainable parameters.  

AdapterBit is proposed to improve downstream SELD scenarios using low-resource data, including limited multi-channel data and even monophonic data. It works better on low-resource data than rich-resource data.
